tools/content-generator/build.gradle.kts

```diff
@@ -18,7 +18,7 @@ import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 
 plugins {
   alias(libs.plugins.nessie.run)
-  id("nessie-conventions-server")
+  id("nessie-conventions-client")
```

Comment:
here we switch conventions the other way around.
as both are basically the same convention now, it may not matter but it looks confusing in the diff.